circuit_knitting/forging/entanglement_forging_operator.py

```diff
@@ -22,9 +22,9 @@
 class EntanglementForgingOperator:  # noqa: D301
     r"""Operator class for Entanglement Forging.
 
-    A class that contains the :math:`2N` qubit Pauli operator :math:`\hat{O} = \sum_{i, j} w_{i, j} \hat{T}_{i, j} \otimes \sum_{a, b} \hat{S}_{a, b}`
-    and associated weights. These operators are knitted by the :class:`EntanglementForgingKnitter` to provide esimates of the
-    energy for the :class:`EntanglementForgingVQE`.
+    A class that contains the :math:`2N` qubit Pauli operator :math:`\hat{O} = \sum_{i, j} w_{i, j} \hat{T}_{i,
+    j} \otimes \sum_{a, b} \hat{S}_{a, b}` and associated weights. These operators are knitted by the
+    :class:`.EntanglementForgingKnitter` to provide estimates of the energy for the :class:`.EntanglementForgingVQE`.
```

Comment:
> * Reference to non-existing `EntanglementForgingVQE` class in `entanglement_forging_operator` (line 27).

`grep` to the rescue: I realized the same identifier mistakenly appears at 
https://github.com/Qiskit-Extensions/circuit-knitting-toolbox/blob/34b9fea053b1784c920c2401fe0a32dbc7a94b7a/test/forging/test_entanglement_forging_ground_state_solver.py#L12

So it looks like we should reference `EntanglementForgingGroundStateSolver` above, and might as well update that line, too.

Reply:
Awesome. This makes sense.